dashboard/app/controllers/pd/workshop_user_admin_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Pd::WorkshopUserAdminController < ApplicationController
+  authorize_resource class: :pd_workshop_user_admin
+
+  # display the Facilitator search form, Facilitators found, and Courses for first Facilitator found
+  # assign / remove Courses actions redirect back to this form to display the updated course list
+  def facilitator_courses_form
+    search_term = facilitator_course_params[:search_term]
+    if search_term =~ /^\d+$/
+      facilitator_id = search_term
+      @facilitators = User.where(id: facilitator_id)
+    elsif search_term
+      email = search_term
+      hashed_email = User.hash_email(email)
+      @facilitators = User.where(hashed_email: hashed_email)
```

Comment:
I'm leaving `@facilitators` in place set with `User.where()` to support the rare use case where there are multiple Users with the same email address.